frontend/packages/ceph-storage-plugin/integration-tests-cypress/views/block-pool.ts

```diff
@@ -0,0 +1,43 @@
+import { NS } from '../utils/consts';
+import { commonFlows } from './common';
+
+// Pool var
+export const poolName: string = 'example.pool';
+export const replicaCount: string = '2';
+export const volumeType: string = 'ssd';
+
+export const populateBlockPoolform = () => {
+  // poolName
+  cy.byTestID('new-pool-name-textbox').type(poolName);
+  // replica
+  cy.byTestID('replica-dropdown').click();
+  cy.byLegacyTestID(replicaCount)
+    .last()
+    .contains(`${replicaCount}-way Replication`)
+    .click();
+  // deviceType
+  cy.byTestID('volume-type-dropdown').click();
+  cy.byTestID(volumeType).click();
+  // compression
+  cy.byTestID('compression-checkbox').check();
+};
+
+export const createBlockPool = () => {
+  commonFlows.navigateToOCS();
+  cy.byLegacyTestID('horizontal-link-Block Pools').click();
```

Comment:
planned to reuse this create function for delete and update block pool testcases